modules/openapi-generator/src/main/resources/typescript-node/models.mustache

```diff
@@ -118,6 +142,17 @@ export class ObjectSerializer {
                 transformedData.push(ObjectSerializer.serialize(datum, subType));
             }
             return transformedData;
+        } else if (startsWith(type, '{ [key: string]: ')) {
+            let subType: string = type.replace('{ [key: string]: ', ''); // { [key: string]: Type; } => Type; }
```

Comment:
I would suggest a regex here. can `Type` be a map again, i.e. contain `{ [key: string]:`?

Reply:
> I would suggest a regex here.

Why a regex? I don't see how that would improve this code at all. Though now that I think about this more a `.slice` would probably be more concise anyway.

> can `Type` be a map again, i.e. contain `{ [key: string]:`?

Yes, which is fine because this code is recursive.